buildSrc/src/main/kotlin/godot/codegen/Class.kt

```diff
@@ -16,6 +16,8 @@ class Class @JsonCreator constructor(
     var baseClass: String,
     @JsonProperty("singleton")
     val isSingleton: Boolean,
+    @JsonProperty("singleton_name")
+    val singletonName: String,
```

Comment:
For singletons, we need to use this property instead of the class name. i.e `_ClassDB` which has a singleton name of `ClassDB`.